src/main/java/ma/hibernate/model/Phone.java

```diff
@@ -4,8 +4,10 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.Table;
 
 @Entity
+@Table(name = "phones")
 public class Phone implements Cloneable {
```

Comment:
"Your task is to implement all PhoneDao methods."
We shouldn't refactor other code.

Reply:
ok)